spec/latest/core/index.html

```diff
@@ -555,9 +555,12 @@ <h3>Documenting a Web API</h3>
 
     <p>Keep in mind that operations specified in an <i>ApiDocumentation</i>
       may fail at runtime as either resources or the <i>ApiDocumentation</i>
-      itself have changed since they have been retrieved. A simple strategy
-      to try to recover from such an error is to reload the
-      <i>ApiDocumentation</i>.</p>
+      itself have changed since they have been retrieved. Also operation details
+      like <i>hydra:returns</i> or <i>hydra:possibleStatus</i> may vary at runtime,
+      thus these interpreted at runtime. A simple strategy to try to recover from
```

Comment:
I think `MUST` is to strong - I'd prefer `SHOULD`.
When the payload is a text, indeed it's quite easy to determine correctness of what we've received, but in case of binary payload (I know - Hydra does not support these yet), it might be tricky to figure it out whether the blob is an AVI movie with DivX encoding we've expected, or an AVI movie, but in h264 encoding. HTTP Content-Type header won't help you much - for both it'll be _video/avi_.